src/lib/useQueryParams.ts

```diff
@@ -109,6 +110,15 @@ export default function useQueryParams(
         delete params.states;
       }
     }
+
+    if (schools !== undefined) {
+      params.schools = Array.isArray(schools)
+        ? Array.from(new Set(schools)).filter((s) => s.length > 0)
+        : [];
+      if (params.schools.length === 0) {
+        delete params.schools;
+      }
+    }
```

Comment:
I actually wouldn't worry about pruning unknown schools (or majors for that matter) since there are more schools and majors than we're aware about.

Reply:
It just prunes if the schools is an empty array or is an array with empty values. Like the majors params.

Reply:
Ah you're right! I forgot about that haha